letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -166,5 +166,7 @@ fi
 # Get the last modified cert directory and copy the cert and private key to store
 # shellcheck disable=SC2012
 CERT_DIR_LATEST="$(ls -td $CERT_DIR/live/*/ | head -1)"
+mkdir "/ssl"
```

Comment:
That directory is bind mounted into the container and essentially guaranteed to exist.